Description: Fix "error: implicit declaration of function ‘strcasecmp’ [-Werror=implicit-function-declaration]"
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: https://github.com/mate-desktop/engrampa/pull/516

--- a/src/file-utils.c
+++ b/src/file-utils.c
@@ -30,6 +30,7 @@
 #include <ctype.h>
 #include <time.h>
 #include <unistd.h>
+#include <strings.h>
 #include <sys/param.h>
 #include <sys/stat.h>
 #include <sys/time.h>
